app/javascript/packs/admin_invoices/content.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function timeToDecimal(time) {
+  const arr = time.split(':');
```

Comment:
`arr` - this is a good variable name for 3rd year student, but not for you of course.